elixir/apps/web/lib/web/live/settings/identity_providers/google_workspace/new.ex

```diff
@@ -41,9 +41,10 @@ defmodule Web.Settings.IdentityProviders.GoogleWorkspace.New do
         Add a new Google Workspace Identity Provider
       </:title>
       <:help>
-        For a more detailed guide on setting up Firezone with Google Workspace, please <.link class={
-          link_style()
-        }>refer to our documentation</.link>.
+        For a more detailed guide on setting up Firezone with Google Workspace, please <.link
```

Comment:
Should we do a `<.docs_ref path="authenticate/google">text</.docs_ref>` component?

Reply:
Good call will add